### ISSUE TITLE
How big about the loss at the beginning of the training？

### ISSUE THREAD

**Issue by ZhengMengbin** [eeskimez/Talking-Face-Landmarks-from-Speech#10]
I use about half an hour of video (slicing video every three seconds) as a training set to train a single announcer (Chinese announcer).  The default training parameters are used, but the test result is that lips are basically not changed. The loss at the beginning of the training is about 0.01, and the loss at the ending of the training is about 0.0007, is it normal？How big about the loss at the beginning of the training of you？And do you know why lips are basically not changed in the test result?
![image](https://user-images.githubusercontent.com/26650043/78159183-cd713c00-7474-11ea-9b4a-546a3bac905b.png)

**Comment by deucalionAlpha** [comment]
I faced a problem similar to yours. The loss was very small from the beginning of training, but the lips of the result video were basically not moving. Did you solve this problem?
